lib/auth/methods.go

```diff
@@ -260,7 +262,7 @@ var (
 	authenticateHeadlessError = trace.AccessDenied("headless authentication failed")
 	// authenticateWebauthnError is the generic error returned for failed WebAuthn
 	// authentication attempts.
-	authenticateWebauthnError = trace.AccessDenied("invalid Webauthn response")
+	authenticateWebauthnError = trace.AccessDenied("invalid credentials")
```

Comment:
Could we backport the message changes? Do you think that's worth it?

Reply:
Yes. I was planning to backport this change along with the UI, but since the UI code in this PR has been updated to be compatible with the new flow, I think I can equally well backport it separately.